src/api/spec/controllers/webui/obs_factory/distributions_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+require 'webmock/rspec'
+# WARNING: If you change owner tests make sure you uncomment this line
+# and start a test backend. Some of the Owner methods
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe Webui::ObsFactory::DistributionsController, type: :controller, vcr: true do
```

Comment:
Why do we need VCR for this?

Reply:
Because it calls the backend for storing the project...

Reply:
Not if you disable the write_through option?